app/src/main/res/values/strings.xml

```diff
@@ -23,42 +23,42 @@
     <string name="parking_free">Free</string>
     <string name="parking_paid">Paid</string>
     <string name="amenities">Amenities</string>
-    <string name="general_info">General information</string>
+    <string name="general_info">General info</string>
     <string name="realtime_data_unavailable">Real-time status unavailable</string>
     <string name="realtime_data_loading">Checking real-time status…</string>
     <string name="realtime_data_source">Real-time status source (beta): %s</string>
     <string name="source">Source: %s</string>
-    <string name="search">Search</string>
+    <string name="search">Search…</string>
     <string name="menu_map">Map</string>
     <string name="menu_favs">Favorites</string>
     <string name="menu_filter">Filter</string>
     <string name="not_implemented">not implemented yet</string>
-    <string name="about">About EVMap</string>
+    <string name="about">About</string>
     <string name="version">Version</string>
     <string name="github_link_title">Source code</string>
-    <string name="oss_licenses">Open Source Licenses</string>
+    <string name="oss_licenses">Libre licenses</string>
     <string name="settings">Settings</string>
-    <string name="settings_ui">User Interface</string>
+    <string name="settings_ui">Interface</string>
     <string name="settings_map">Map</string>
     <string name="copyright">Copyright</string>
     <string name="copyright_summary">©2020–2022 Johan von Forstner</string>
     <string name="other">Other</string>
-    <string name="privacy">Privacy Notice</string>
-    <string name="fav_add">Add to favorites</string>
-    <string name="fav_remove">Remove from favorites</string>
-    <string name="pref_navigate_use_maps">Start navigation immediately</string>
-    <string name="pref_navigate_use_maps_on">Navigation button starts Google Maps navigation immediately</string>
-    <string name="pref_navigate_use_maps_off">Navigation button launches maps app with charger location</string>
+    <string name="privacy">Privacy</string>
+    <string name="fav_add">Favorite</string>
+    <string name="fav_remove">Un-favorite</string>
+    <string name="pref_navigate_use_maps">Immediate navigation</string>
+    <string name="pref_navigate_use_maps_on">The nav button opens Google Maps</string>
```

Comment:
```suggestion
    <string name="pref_navigate_use_maps_on">Opens your map app directly from the nav button</string>
```

Reply:
No, what the string needs to state is that if the setting is on, it will not only open Google Maps, but start navigation/route guidance immediately. See my comment above.

Reply:
Better now?